### ISSUE TITLE
I can't update my blocks

### ISSUE THREAD

**Issue by Singhol** [jsha/blocktogether#294]
I've created this list for a week now, but it's been a few days that I can't get it updated

**Comment by jsha** [comment]
Thanks for the note. I was doing some maintenance a few days ago and forgot to restart a script. Should be better now.

**Comment by sxe** [comment]
Hi, 
didn't want to open a new issue for this but does the site still having problems?
I have two twitter accounts and I use blocktogether to sync blocks between them. It's now day 11 without updates either direction. 

Cheers

**Comment by jsha** [comment]
I've made a couple more fixes and the site is now working through a backlog. If you want to share your twitter handle I can check on it and see if there are any additional problems.

**Comment by sxe** [comment]
Hey, thanks for the offer but if there is a backlog that's probably the reason. I will wait a couple more days and see if it catches up. If not I will let you know. 

Cheers

**Comment by DhotyaShield** [comment]
My list hasn't updated in 12 days. Thanks for the work you do on this jsha. Twitter handle is as username here @DhotyaShield

**Comment by Willt125** [comment]
New voice here. It's only been a couple days here (since I first joined!), but just informing that it's not grabbing any of my blocks either.

**Comment by sxe** [comment]
Yeah it's 17 days for me now in either direction.

**Comment by Scott-Tracy** [comment]
Same here, 17 days, although it appears that new blocks are propagating to subscribers.

**Comment by sxe** [comment]
@Scott-Tracy what exactly do you mean then when you say "Same here, 17 days"? 
What I mean. Blocks didn't get shared at all since now 18 days.

**Comment by shadowblocker** [comment]
We've added at least 500 blocks over the last week, none of which are showing in blocktogether, and none of the subscribers have received the new blocks. The MyBlocks page is showing no updates for 5 days, the actions page hasn't update for 19 days, the subscribers page doesn't appear to have updated either despite a few users saying they have subbed.

Is there any specific diagnostic info we can provide to help get to the bottom of this?

**Comment by sxe** [comment]
@jsha are you still working on this? It's not 25 days since the last "action". I don't want to put pressure on you or anything, just some update on your progress would be nice.

**Comment by jsha** [comment]
Yep, I'm still working on this as I have time. Haven't had a ton of time
lately.

I did find at least one source of the problems last night: after a recent
upgrade, one of the processes was running out of memory and crashing. I
need to figure out why it wasn't automatically restarting, but for now it's
uo and running again.

**Comment by sxe** [comment]
Thanks for the feedback. Lets see if it catches up. Right now I am still stuck at 26 days.

**Comment by 7adietri** [comment]
It took a while after jsha's message, but I'm seeing blocks now!

**Comment by 7adietri** [comment]
It seems to be stuck again, both "My Blocks" and "Actions" aren't updating.

**Comment by jsha** [comment]
Posted an update on the blocktogether Twitter account: https://twitter.com/blocktogether/status/1227804407506407424

**Comment by 7adietri** [comment]
I'm currently seeing `updated 2 days ago`, so it might be stuck again. Worked great before that, though.

**Comment by Scott-Tracy** [comment]
FYI: List updates have never worked as well for me than in the past week, new blocks and the associated actions are showing up in real time. Fabulous!
